src/components/dar_collection_table/DarCollectionTableCellData.js

```diff
@@ -55,6 +55,7 @@ export function darCodeCellData({darCode = '- -', darCollectionId, collectionIsE
       h((collectionIsExpanded ? ExpandLess : ExpandMore), {
         id: `${darCollectionId}_dropdown`,
         className: `sort-icon dar-expand-dropdown-arrow ${collectionIsExpanded ? 'sort-icon-up' : 'sort-icon-down'}`,
+        isRendered: status !== 'Draft',
```

Comment:
To be safe, use lodash's `toLower` on `status` so that you don't have to worry about casing